### DISCUSSION TITLE
Timeout Error

### DISCUSSION THREAD

**Discussion by JSab01** [GerryDazoo/Slinger#622] [Q&A]
New to this, so I apologize in advance. I was able to get this working yesterday and then it stopped. I am getting the following timeout error while trying to connect. Port forwarding is open and aligned to the server computer. It does change though, which is quite annoying.

Slinginfo  M1 12 30 xxx
63999 Starting Stream for  xxx
63999 Opening Control stream 0x2000 0 0
Connecting... ('IP, Port) Control
Traceback (most recent call last):
  File "slingbox_server.py", line 836, in streamer
    s_ctl, stream  = start_slingbox_session(streams)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "slingbox_server.py", line 359, in start_slingbox_session
    s_ctl = sling_open(sling_net_address, 'Control') # open control connection to SB
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "slingbox_server.py", line 320, in sling_open
    s.connect(addr)
TimeoutError: timed out
63999 Badness starting slingbox session  timed out None
Streamer:  63999 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox

**Comment by GerryDazoo** [comment]
you need to post the complete logs including the server startup and your config.ini. But from what yove posed it sure looks like you do not have a valid IP address/port number for your slingbox in your config file.

**Comment by JSab01** [comment]
Here is my full config code:

[SLINGBOX]
sbtype=M1
;sbtype="Solo/Pro/ProHD"
;sbtype="240"
password=E1:[filled in with my password]
;ipaddress=192.168.1.8 [this is where my router says my slingbox M1 is sitting]
;port=5310 [this is the port that shows up in the sling server log]

;Replace with your finderid, If and only if you need remote access and don't  
;have a static ip and you plan on using my service to access you server remotely. 
;Please read the release notes. Most people don't need this.
;finderid=BAD95DAED609364C8204F2BC5FBDC363

;If you've got more than one slingbox set name. THe server will use this when generating 
;logs to make it easier to see what's going on. 
;name=something

;Following are the default values
;Default resolution when server starts. You can change this and reconnect without having to restart the server
;valid range = 0..16 
Resolution=12

;Valid Values 1, 6, 10, 15, 20, 30, 60   depending on resolution
FrameRate=30

;Min 50 Max 8000
VideoBandwidth=2000

;Min 0 (auto) max 63
VideoSmoothness=63

;Send Iframe every n seconds Max 30
IframeRate=5

;Audio Bit Rate. Valid Options 16, 20, 32, 40, 48, 64, 96 Default 32
AudioBitRate=32

;Video Source  0, 1, 2, 3 depending on your hardware corresponds to one of
;Composite, Component, S-Video, HDMI or Tuner.
;Pro:        0=(Analog)Tuner   1=Composite   2=S-Video     3=Component (Via HDMI to Component Cable)
;ProHD:      0=Tuner           1=Composite   2=S-Video     3=Component
;Solo:       0=Composite       1=S-Video     2=Component
;500:        0=Composite       1=Component   2=HDMI
;350/M1/M2:  0=Composite       1=Component
; If you don't set this, the code will use the last configured input
; I recommend setting this value. If not set correctly, you'll often only see a black screen.
; Note: Setting invalid options may cause unexpected hardware behavior. For example, setting
;  2 or 3 for M2 hardware causes it to reboot.
VideoSource = 1

[SERVER]
;local port number for the server to listen on for connections
port=63999 [this is the port that I have opened]
maxstreams=10
enableremote=yes

[REMOTE]
;see release notes for changing this if your remote doesn't work, But as a start make it 
;the same as your configured VideoSource 0-3. See above.
code=1
;path to an external remote control definitions file, if the defaults
;aren't working for you or you want to change the format of the web page.
;use the supplied remote.txt as a starting point.
include=remote.txt 

attempting to get the full log from a remote desktop.

**Reply by GerryDazoo** [reply]
you have the IP address and port number commented out. remove the leading ";"

**Comment by JSab01** [comment]
![IMG_7950](https://github.com/GerryDazoo/Slinger/assets/145022617/1cba0d77-d852-4d28-a117-1b21e7d14a5c)

thank you, still not getting there.
